packages/core/src/services/documentLogs/computeDocumentLogsDailyCount.ts

```diff
@@ -13,15 +17,27 @@ export type DailyCount = {
 export async function computeDocumentLogsDailyCount(
   {
     documentUuid,
-    draft,
+    filterOptions,
+    workspace,
     days = 30,
   }: {
     documentUuid: string
-    draft?: Commit
+    filterOptions?: DocumentLogFilterOptions
+    workspace: Workspace
     days?: number
   },
   db = database,
 ): Promise<DailyCount[]> {
+  const conditions = [
+    eq(projects.workspaceId, workspace.id),
+    isNull(runErrors.id),
+    sql`${documentLogs.createdAt} >= NOW() - INTERVAL '${sql.raw(
```

Comment:
Is sql.raw necessary? Could it lead to sql injections?

Reply:
No sql injections because is inside `sql`